ramls/pick-slips-report.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Success response for pick slip report generation",
+  "properties": {
```

Comment:
Is it expected that report contains only one record?

Reply:
It seems that you need an array of items here.

Reply:
thanks, fixed